cns/wireserver/net.go

```diff
@@ -49,6 +52,48 @@ func GetPrimaryInterfaceFromResult(res *GetInterfacesResult) (*InterfaceInfo, er
 	return nil, ErrNoPrimaryInterface
 }
 
+// Gets secondary interface details for swiftv2 secondary nics scenario
+func GetSecondaryInterfaceFromResult(res *GetInterfacesResult, macAddress string) (*InterfaceInfo, error) {
+	for _, i := range res.Interface {
+		// skip if primary
+		if i.IsPrimary {
+			continue
+		}
+
+		// skip if no subnets
+		if len(i.IPSubnet) == 0 {
```

Comment:
What is the logic here? Can we add a bigger comment?

Reply:
this is a replica of the aboce func - `GetPrimaryInterfaceFromResult` to find similar details about secondary nics. for the interfaces in list, if subnet does not exist should skip the nic incase it's not programmed or usable for IP

Reply:
addressed